datastore/storage/boltdb.go

```diff
@@ -0,0 +1,151 @@
+package storage
+
+import (
+	"errors"
+	"os"
+	"strconv"
+
+	"github.com/boltdb/bolt"
+)
+
+type DatastoreBoltDB struct {
```

Comment:
Perhaps name this `BoltDBStorage` to match the naming of `MemStorage` type?

Reply:
done

Reply:
Note comment about naming.